ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SyslogIntegrationForm.tsx

```diff
@@ -53,6 +53,14 @@ export const validationSchema = yup.object().shape({
             useTls: yup.bool(),
             skipTlsVerify: yup.bool(),
         }),
+        maxMessageSize: yup
+            .number()
+            .required('Message size is required')
+            .test(
+                'message-size-test',
+                'Message size must be between 0 and 1048576',
+                (value = 0) => value >= 0 && value <= 1048576
```

Comment:
I don't think we should be adding an explicit default parameter for this callback function.

It should not matter as long as the `required` and `number` checks are before it in the call chain, but just in case either of those are removed later, then this may force an empty string to be passed into the checking function as `0` which would then always pass the check.